scheduler/filter/filter.go

```diff
@@ -33,6 +33,7 @@ func init() {
 		&SlotsFilter{},
 		&DependencyFilter{},
 		&AffinityFilter{},
+		&WhitelistFilter{},
 		&ConstraintFilter{},
```

Comment:
Ah! My bad actually, I commented before reading through the rest of the code. I misunderstood what this filter was supposed to do (whitelist to me implied that it's a whitelist of nodes indicating that regardless of the outcome of other filters, a whitelisted node should always be allowed to run the task).

I now understand, and sorry for the confusion.